routes/products.js

```diff
@@ -11,9 +12,9 @@ let db = admin.firestore();
 
 function productsApi (app){
   const router = express.Router();
-  app.use('/api', router);
```

Comment:
You are adding `api` to all of the endpoints except for the Ads and Medical ones.
You should keep the prefix on all the routes.